packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/MrwSerializationTypeDefinition.cs

```diff
@@ -1401,6 +1407,24 @@ private MethodBodyStatement CreateListSerializationStatement(
             };
         }
 
+        private MethodBodyStatement CreateSerializationStatementForReadOnlyMemory(ScopedApi array, SerializationFormat serializationFormat)
```

Comment:
Any way to avoid duplicating some of this code? I think we just need the special handling for accessing the Span property but everything else looks to be the same as the non-ROM case.

Reply:
I've consolidated this method.